src/bundle/Resources/public/js/scripts/admin.back.to.top.js

```diff
@@ -8,18 +8,35 @@
     }
 
     const backToTopBtnTitle = backToTopBtn.querySelector('.ibexa-back-to-top__title');
-
-    backToTopScrollContainer.addEventListener('scroll', (event) => {
-        const container = event.target;
+    let currentBackToTopAnchorHeight = backToTopAnchor.offsetHeight;
+    const toggleBackToTopBtnText = (container) => {
         const isTitleVisible = Math.abs(container.scrollHeight - container.scrollTop - container.clientHeight) <= 2;
 
         backToTopBtn.classList.toggle('ibexa-back-to-top__btn--visible', container.scrollTop !== 0);
         backToTopBtn.classList.toggle('ibexa-btn--no-text', !isTitleVisible);
         backToTopBtnTitle.classList.toggle('ibexa-back-to-top__title--visible', isTitleVisible);
+    };
+
+    backToTopScrollContainer.addEventListener('scroll', (event) => {
+        const container = event.target;
+
+        toggleBackToTopBtnText(container);
     });
     backToTopBtn.addEventListener('click', () => {
         backToTopAnchor.scrollIntoView({
             behavior: 'smooth',
         });
     });
+
+    const resizeObserver = new ResizeObserver((entries) => {
+        if (currentBackToTopAnchorHeight === entries[0].target.clientHeight) {
+            return;
+        }
+
+        currentBackToTopAnchorHeight = entries[0].target.clientHeight;
+
+        toggleBackToTopBtnText(backToTopScrollContainer);
```

Comment:
`toggle` in the name of the function suggests that this function will show/hide back-to-top-button on resize event. But it is actually checking inside whether it should do this which is not intuitive.